redbot/cogs/mod/settings.py

```diff
@@ -80,6 +83,9 @@ async def modset_showsettings(self, ctx: commands.Context):
             )
         else:
             msg += _("Default message history delete on ban: Don't delete any\n")
+        msg += _("Tempban default: {duration}").format(
```

Comment:
Maybe this would sound better?
```suggestion
        msg += _("Default duration of a tempban: {duration}").format(
```